### PR TITLE
Re-Adds exosuits to RnD, without the runtimes hopefully.

### PR DESCRIPTION
**Pull request by githubuser4141**
## About The Pull Request

It's mostly a copy-paste of #193 but hopefully works.

## Why It's Good For The Game

See #193 

## Pre-Merge Checklist

- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

:cl:
add: Does what #193 did.
del: Removes the pipe machinery as it was causing runtimes in RnD somehow. Fixes weapon firing pins causing RnD runtimes. Mech weapons no longer use the deprecated electronic_weapons RnD category. 
balance: Adds a beepy sound for when mechs' HP goes below a certain number. Increases explosion chance.

/:cl:

### CODE REVIEW

File: code/game/mecha/combat/durand.dm [hunk 1]
```diff
@@ -1,17 +1,209 @@
...
+	shield = new/obj/durand_shield
+	shield.chassis = src
+	shield.layer = layer
+	RegisterSignal(src, COMSIG_MECHA_ACTION_ACTIVATE, .proc/relay)
+	RegisterSignal(src, COMSIG_PROJECTILE_PREHIT, .proc/prehit)
+	. = ..()
+
+/obj/mecha/combat/durand/Destroy()
+	if(shield)
+		qdel(shield)
```
Comment (RonBag) [inline code/game/mecha/combat/durand.dm:29]: QDEL_NULL(shield)

File: code/game/mecha/combat/durand.dm [hunk 2]
```diff
@@ -1,17 +1,209 @@
...
+	var/switching = FALSE ///To keep track of things during the animation
+	anchored = TRUE
+
+/obj/durand_shield/Initialize()
+	. = ..()
+	RegisterSignal(src, COMSIG_MECHA_ACTION_ACTIVATE, .proc/activate)
+
+/obj/durand_shield/Destroy()
+	if(chassis)
+		chassis.shield = null
```
Comment (RonBag) [inline code/game/mecha/combat/durand.dm:146]: add chassis = null

Comment (githubuser4141) [reply code/game/mecha/combat/durand.dm:146]: Sorry, what's add chassis supposed to do? I don't know a whole lot and add. or add throws an error.

Comment (RonBag) [reply code/game/mecha/combat/durand.dm:146]: ah, my bad. add this specific line without the two "
"chassis = null"

Comment (githubuser4141) [reply code/game/mecha/combat/durand.dm:146]: > ah, my bad. add this specific line without the two " "chassis = null"

So.. I changed it to that and afterward it'd runtime whenever the shield was destroyed and get 'stuck' so it couldn't be turned on/off again.

Comment (RonBag) [reply code/game/mecha/combat/durand.dm:146]: > > ah, my bad. add this specific line without the two " "chassis = null"
> 
> So.. I changed it to that and afterward it'd runtime whenever the shield was destroyed and get 'stuck' so it couldn't be turned on/off again. Wait, is it supposed to be l 145 or 146?

Add it after line 146. 
So, 
"/obj/durand_shield/Destroy()
	if(chassis)
		chassis.shield = null
                chassis = null
"
*rest of the code*

Comment (githubuser4141) [reply code/game/mecha/combat/durand.dm:146]: Think I got it, thanks. Though it does cause a runtime when admin induced things happen (e.g. lowering the shield's HP to 1 and hitting it.)

**Pull request merged**